deploy/roles/stata/tasks/main.yaml

```diff
@@ -8,6 +8,18 @@
   tags:
     - stata
 
+- name: Ensure no installed files exist
+  file:
+    path: "{{ stata_install_directory }}/{{ item }}"
+    state: absent
+  with_items:
+    - installed.150
+    - stata
+    - xstata
+    - isstata.150
+  tags:
+    - stata
```

Comment:
@rioh What about the license file, stata.lic?  If that is there, the installer prompts will change which will interfere with the expect script, no?  Or does removing these 4 files prevent that check from even happening?

Reply:
Yes, I think the installed.150 prevents it from running that check. We can remove the license too, it won't hurt.